client-lite/src/util/do_file.cpp

```diff
@@ -0,0 +1,57 @@
+#include "do_common.h"
+#include "do_file.h"
+
+#include <fcntl.h>
+#include <sys/stat.h>
```

Comment:
Cool, the usage of posix apis potentially allows us to cut a little bloat (boost::filesystem) from our components